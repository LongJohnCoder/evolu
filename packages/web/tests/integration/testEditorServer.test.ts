import { pageDom, pageGoto } from './helpers';

beforeEach(async () => {
  await pageGoto('testEditorServer');
});

test('render', async () => {
  await expect(await pageDom()).toMatchSnapshot();
});
