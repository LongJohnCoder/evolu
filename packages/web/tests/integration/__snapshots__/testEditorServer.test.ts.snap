// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  id="__next"
>
  <div>
    <div>
      <div
        class="root"
      >
        <div
          style="font-size:24px"
        >
          ·heading·
        </div>
        <div
          style="font-size:16px"
        >
          ·paragraph·
        </div>
      </div>
    </div>
    <div>
      <div
        class="css-0"
      >
        <div
          class="css-vlbfxt-heading"
        >
          ·heading·
        </div>
        <div
          class="css-1yjna00-paragraph"
        >
          ·paragraph·
        </div>
        <div
          class="css-1n9mwyn-list"
        >
          <div
            class="css-0"
          >
            ·listitem·
          </div>
        </div>
        <img
          alt="Square placeholder image 80px"
          height="80"
          src="https://via.placeholder.com/80"
          width="80"
        />
      </div>
    </div>
    <div>
      <div>
        ·ab·
        <br />
        ·c·
      </div>
    </div>
  </div>
</div>
`;
